spacy/training/loggers.py

```diff
@@ -84,6 +107,13 @@ def setup_printer(
             write(msg.row(table_header, widths=table_widths, spacing=spacing))
             write(msg.row(["-" * width for width in table_widths], spacing=spacing))
         progress = None
+        expected_progress_types = ("train", "eval", None)
+        if progress_bar not in expected_progress_types:
+            raise ValueError(
+                Errors.E1048.format(
+                    unexpected=progress_bar, expected=expected_progress_types
+                )
+            )
```

Comment:
I find it a bit odd to add `None` to the printed types here, instead I'd do 
```
if progress_bar is not None and progress_bar not in expected_progress_types
    ....
```

Reply:
Done.